dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1638,6 +1649,9 @@ private void initializeValidations(String include, Workflow workflow) {
         if (checkIncludes(include, ALIASES)) {
             workflow.getWorkflowVersions().forEach(workflowVersion -> Hibernate.initialize(workflowVersion.getAliases()));
         }
+        if (checkIncludes(include, IMAGES)) {
```

Comment:
update comments in this function. probably rename function too

Reply:
renamed